### PR TITLE
Move code-studio entry points into new directory structure.

### PR DESCRIPTION
**Pull request by pcardune**
This is part 1 of the first step in the refactoring plan defined in this doc: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/docs/refactor.md#the-actual-plan.

There are more entry points to move, and some of these entry points could be combined and/or refactored. But I'd like to reduce the number of changes in a single PR.

### CODE REVIEW

File: apps/src/sites/studio/pages/code-studio.js [hunk 1]
```diff
@@ -12,57 +12,57 @@ require("babel-polyfill");
...
-require('./components/SendToPhone');
-require('./components/small_footer');
-require('./components/GridEditor');
-require('./components/Attachments');
+require('@cdo/apps/code-studio/components/abuse_error');
+require('@cdo/apps/code-studio/components/report_abuse_form');
+require('@cdo/apps/code-studio/components/SendToPhone');
+require('@cdo/apps/code-studio/components/small_footer');
+require('@cdo/apps/code-studio/components/GridEditor');
+require('@cdo/apps/code-studio/components/Attachments');
```
Comment (Bjvanminnen) [inline apps/src/sites/studio/pages/code-studio.js:31]: Is there a rule for when we'd like to use @cdo/apps vs. a relative reference for paths?

**Comment by Bjvanminnen** [comment]
changes lgtm. like the idea of doing things in small chunks like this as much as possible :)

File: apps/src/sites/studio/pages/code-studio.js (hunk 1 above)
Comment (pcardune) [reply apps/src/sites/studio/pages/code-studio.js:31]: There isn't one yet! But I like the idea of only using relative paths for subdirectories. i.e. `../*` would not be allowed, only `./*`

What do you think of that idea?

Comment (Bjvanminnen) [reply apps/src/sites/studio/pages/code-studio.js:31]: My initial reaction is that this might be too restrictive, though I could be wrong about that.

**Pull request merged**